ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods.yaml

```
@@ -0,0 +1,62 @@
+base_images:
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  rhods-addon-tests:
+    name: ods-ci
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.12"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rhods-tests
+  cron: 0 12 * * 6
+  remote_api: true
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: managed-odh|notification-email=interop-qe-ms@redhat.com
```

```suggestion
      ADDON1_CONFIG: "managed-odh|notification-email=interop-qe-ms@redhat.com"
```
common practice - please wrap all with double quotes

---

done